StripeConnect/StripeConnect/Source/EmbeddedComponentManager+Appearance.swift

```diff
@@ -103,7 +103,12 @@ extension EmbeddedComponentManager {
             public var labelMd: Style = .init()
             /// Describes the font size and weight for the small label typography.
             public var labelSm: Style = .init()
-            
+
+            /// Treats the given font sizes as unscaled sizes and scales the
+            /// appearance when the component's trait collection is updated
+            /// - Note: This is an experimental feature that may be removed at any time.
+            @_spi(ExperimentalFontScaling) public var automaticallyScaleFontSize = false
```

Comment:
Chose to use `ExperimentalFontScaling` instead of `DashboardOnly` because it hasn't gone through API review and needs to be SPI-protected, but we don't want integrators to import with `DashboardOnly` just to get access to it.

If it works well for Dashboard, we'll likely put it through API review and expose publicly.